experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -144,7 +81,8 @@ function make_lsm_domain(
 end
 
 # required by Interfacer
-get_field(sim::BucketSimulation, ::Val{:surface_temperature}) = sim.integrator.p.bucket.T_sfc
+get_field(sim::BucketSimulation, ::Val{:surface_temperature}) =
```

Comment:
Why do we need both this and `get_land_temp_from_state`?

Reply:
This is required by the `viz_explorer.jl` but we do want to remove this eventually. I've already opened an [issue](https://github.com/CliMA/ClimaCoupler.jl/issues/332) to generalize these functions.